app/lib/rocdev/chat/chat_inviter.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Rocdev.ChatInviter do
+  @moduledoc """
+  Provides access to Slack API.
+  """
+
+  use Tesla
```

Comment:
ehhhhhhhh this is a style thing but I don't love the `use` pattern for stuff like this.  This could just as easily be a simple function call.  Instead it's got a ton of indirection.  Elixir ain't Ruby.

Ultimately your choice, but if you're asking me I would maybe be looking at a different HTTP client.

Reply:
Any suggestions? One thing I like about this library is how easy it is to test. The only reason I haven't yet is because I'm playing with Mountebank and wanted to get this in.

Reply:
HTTPoison and HTTPotion are the "standard"s, though I'm not sure on their testability.  It's possible you can use Tesla without `use`ing Tesla?

Reply:
Looking more deeply into HTTPoison, they also seem to recommend the `use HTTPoison.Base` for this use case. /shrug